test/models/track_test.rb

```diff
@@ -16,6 +16,14 @@ class TrackTest < ActiveSupport::TestCase
     assert_equal track, Track.for!(track.slug)
   end
 
+  test "#active scope" do
+    # Create one active and one inactive track
+    track = create :track, active: true
+    create :track, active: false
```

Comment:
Ha - And Erik perfectly emphasises my point re the comment. It would be really easy for Erik to remove this line because it seemingly does nothing (although in reality, it's crucial to this test working).